src/arch/riscv/machine/hardware.c

```diff
@@ -167,6 +168,10 @@ static irq_t handleVTimer(void)
 
 static irq_t active_irq[CONFIG_MAX_NUM_NODES];
 
+static inline void sbi_clear_ipi(void)
```

Comment:
Please remove this, this function is already defined in `include/arch/riscv/arch/sbi.h`, see https://github.com/seL4/seL4/blob/master/include/arch/riscv/arch/sbi.h#L101-L104

Reply:
Done.